pysgrid/sgrid.py

```diff
@@ -419,29 +448,102 @@ def locate_faces(self,
         else:
             res = np.ma.masked_less(ind, 0)
             if _memo:
-                self._add_memo(points, res, grid,
-                               self._ind_memo_dict, _copy, _hash)
+                self._add_memo(points, res, grid, self._ind_memo_dict, _copy, _hash)
             return res
 
+    def locate_nearest(self,
+                       points,
+                       grid,
+                       _memo=False,
+                       _copy=False,
+                       _hash=None):
```

Comment:
They are there to present a consistent API for locate_ functions, and are likely to be used in future.